trajopt/src/problem_description.cpp

```diff
@@ -208,16 +234,17 @@ void ProblemConstructionInfo::readInitInfo(const Json::Value& v)
 {
   std::string type_str;
   json_marshal::childFromJson(v, type_str, "type");
+  json_marshal::childFromJson(v, init_info.dt, "dt", 1.0);
   int n_steps = basic_info.n_steps;
   int n_dof = kin->numJoints();
-  Eigen::VectorXd start_pos = env->getCurrentJointValues(kin->getName());
 
-  if (type_str == "stationary")
+  if (boost::iequals(type_str, "stationary"))
```

Comment:
Should we use `type_str.compare("stationary") == 0` instead of boost?

Reply:
This is something Reid had in there so that it would be case insensitive. I'll go with whatever you recommend if there's a better way of doing this (or if you'd prefer it to be case sensitive)